### DISCUSSION TITLE
Can i pass slug params?

### DISCUSSION THREAD

**Discussion by Martian95** [directus/directus#19235] [Feature Requests]
### Summary

can i pass the slug and part of the id in a long string to get the specific details of a post. for example GET /items/posts/this-is-title-449d8e55 instead GET /items/posts/449d8e55-4d15-4398-a7bf-32b6c9b6b2a9?

### Basic Example

can i pass the slug and part of the id in a long string to get the specific details of a post. for example GET /items/posts/this-is-title-449d8e55 instead GET /items/posts/449d8e55-4d15-4398-a7bf-32b6c9b6b2a9?

### Motivation

I think it will be better for SEO for websites that consider SEO important

### Detailed Design

i think you should add

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

no

### Alternatives

no

### Adoption Strategy

no

**Comment by DanielBiegler** [comment]
Your frontend is responsible for SEO, not the raw API. This means your frontend has an URL similar to `example.com/post/:slug` where slug can be defined by you in the API. See the docs for examples: https://docs.directus.io/guides/headless-cms/build-static-website/nuxt-3.html#creating-blog-posts-with-directus
